Misc/NEWS.d/next/Library/2024-12-11-17-44-36.gh-issue-127794.VwmRsp.rst

```diff
@@ -0,0 +1,4 @@
+The :meth:`email.message.Message.add_header` method now validates header
+field names according to :rfc:`5322` and raises a :exc:`ValueError` if
```

Comment:
Can we have the dedicated section with field names specifications? If you find it, you can link it as <code>:rfc:\`5322#section-X.Y\` and this would users knowing about it instead of going through the entire RFC.

Reply:
Addressed in 56a77e93524